packages/main/src/components/sessions-signout/AllSessionSignOut.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import { useHistory } from "react-router-dom";
+import styles from "../styles/AllSessionSignOut.module.css";
+import SignOutActionList from "./SignOutActionList";
+import SettingsNav from "./SettingsNav";
+import FormMessage from "./FormMessage";
+import boxArrow from "./../box-arrow-right.svg";
+import errorIcon from "../../../component-assets/icons/bx-error.svg";
+import axios from "axios";
+
+function AllSessionSignOut() {
+  const [error, setError] = useState("");
+  //const [userSession, setUserSession] = useState({})
+  const [user, setUser] = useState();
+  const [currentPassword, setCurrentPassword] = useState("");
+
+  const history = useHistory();
+
+  useEffect(() => {
+    if (
+      sessionStorage.getItem("user") &&
+      sessionStorage.getItem("session_id")
+    ) {
+      const userInfo = JSON.parse(sessionStorage.getItem("user"));
+      // console.log(userInfo)
+      setUser(userInfo);
+    } else {
+      history.push("./login");
+    }
+  }, [history]);
+
+  useEffect(() => {
+    let timerId;
+    if (error) {
+      timerId = setTimeout(() => {
+        setError("");
+      }, 3000);
+    }
+
+    return () => clearTimeout(timerId);
+  });
+
+  const handleSubmit = async e => {
+    e.preventDefault();
+
+    const passwordData = {
+      password: currentPassword
+    };
+
+    try {
+      const response = await axios.post(
+        "https://api.zuri.chat/auth/logout/othersessions",
```

Comment:
There are different environments, use the environment variable for this.

Reply:
Please I don't understand by environment variable....can you please add more context?
@billmal071

Reply:
There's an environment variable to get the base url for the api depending on if it's on dev, staging or main(prod).